embark/templates/uploader/individualReportDashboard.html

```diff
@@ -1,6 +1,13 @@
 {% extends "uploader/home.html" %}
 {% block maincontent %}
 
+<style>
+tbody tr td:first-child {
+  float: left;
+  min-width: 350px;
+}
+</style>
```

Comment:
Better: Add a class/id to the table, to which the style should be attached and move this code into one of the imported css files.